tutorials/FACTS.ipynb

```diff
@@ -0,0 +1,756 @@
ccP11l8EAp1eKusuSXNE/L0zWSDYH6j1rFRYWMmfOHBYtWgTAhQsXaN++Pc899xxt2rThtddeM/sgb2nK0xlTz4nFo2czV2b7+99iQtS4iBQQlZVpx/V7Shhmx38R3smrFbIO/yh6lvn1B98mSito4Vw6LFLjWicko7C1xW3mw/U+h0FvICVWGGaD7ulA4oUs9q28xKF10Th5amjTWXiNS+pQZiyNFbVb1gHtyEouBGDA3UG0aic8lrc91pVVHx8lK7mQP78/TUB3D45tjkVbqgcFuPs40Lq9M5lX8km6lENWUgEAjm0DUFw+j0GpoCA7C0d3Y0GyQiE+iwWpRsn8GryOBgP8+ar4jAWPq95HzL8f9HwITi4Wk+gTO8Wk2hDK+xx1uavqZ/xqVFYw7NWGnf8WRs5NFsLZR6Q1DnkFLm6B5HARBSgrEtLzKrVIsbJ2EJ95ew+RIuzgKSIGBWmQnyqO2/OpECAoyYMJXzW8x9SVk/DLpEqj7KFVjUs1VKpg8jzRhsWlrXAQmtLfTHLDKXdCBvbuh7WmYT2rFAoFHQcO5cjalUTs2900hhkIyf52Q4Ta9JmVYo4BEU0b95Go9zIleqxQCAfid/2FmNqJxRA6vfb9k8/AplcrU+ZdA2DS9w2f1yRmpd4uqddff51Tp06xc+dONJpKydHRo0fz+++/m3VwEkQqVnkD5vJFXXPl4rZKae+7vq3MoTaFDmNEyL44R3SbL6ckD47MF68HmqYQKKlKcX4+ydHCA+eVW4jrtGm1inPURXZqISWFWtRWStx87Ok52p+QkWJxcuFwSoUyY+nly+jyC2o8R3kqI77tKMwtBcCltV3FdhtbtUhPtFOTEpPLoXXRaEv1eAc5M/X1MO77V1+GP9CR9j09AchMEsadTVt/bLRCmbHekvnn1gpvo8oGxv2n5n1GvyNSi5PDRT5+QygrFlLdYPkeOi0QOTdZGJVaCNgMfw1u+7foGznpWyGKMfYDIWA16DkRaeowGrxDRFS47UARGR71lmiBolDBySWw6lGRgVFfEo8bI2XZ4NsXHlxpnvovJ2944He4/b/SKGum6PU6Ig+INNNr0xjrS6eBovdc9IkjlBTWPJ+ZBac2cO98eHgDBAwRTurZh0X9fX1Sel3bCqc2iChYflr1fcqKRZr8D0OFUaa2FaUmzxy6eUplbgHqbZitWbOGb775hsGDB1dpOtulSxeioqLqOFLSYLrdI573fgbr5ghjpbmRlwJ/PCleh82CznfW73ilEvoZjz/0g4hiZMfDwvFignVtJxTrJPUm/mw4GAw4FJdiq1Th9kjD1AVTjGmMnm0dUanErSMwVBhIaXF5qN3cULcWjcJLImqOmpUbZkWuYmFj52SNjW1Vr7iLlx1jn+iGykqJvYsNtz3WhbtfDsXTv3Jx5eotok2ZxoiZlZ8fGqMyY3XDzEU81ySZX5wLf/2feD34hdrTkhw8Rf49iF4v2pKa96uLC5uF48HJR0zAErMi56YWQMh9oo2Kylo4TJbPEFLgpnJxK/x8Z6VR9tCq6hL3khZLwrmzFGRlYmNvT0BILX3nTMSzbTvcff3RlZVx6chBM42wDtoNESmI4z9uuOp0/2egdXfx+d/8mlhHlRN/RBhkez8XUvtd7oJnjwgBEdnjtVlRb8MsLS0NLy+vau8XFBRUmQwlZqTHNFF3gkKk+s0dKHrH6HXV99WWCInV2H0i53/Lv0Sflb/+D44uFHLgTdFs989XRVF3q27Va8RMpeeDIt0lPRJ2fyJyrpNPiyaG9/wkw+wN5PLpEwC45xXicu89WLWq/v01hWSjYdaqXWVtmoefIwoFFOaUUpBTgm33bgAUHj9R7XiDTkfp5Tix3UakGrq0squ2H4BfJzdm/mcQ098fQHBY62r3FjejYZaTUohOp8fa37+il1lu2jXKjI7CWOTgXCFAUE5poWjyXN5Q83pNOvs8Co7eUJQJF/6qe9+aCDdGbbpPkZ/lJkDOTS2EzhNEPZhaAxf+hNWzap7rruX4L+L7XFYg6t6mr5ZG2S1GxH6RxhjcbxBqKxPLKGpBoVBURM3KVR6bPSq1SAFWKEVq5Pte8GUI/HSbqJNOjxRpxNOWCCVRGfltltTbMAsLC2Pjxo0VP5dPePPmzWPAgAHmG5mkEqVSpILM3CB6WWTHid4W/3aD/2fvvMPbKq8//rnakiXvvWInzt5kskISQggjEHYZZZS2QCEt0NJCB1AoBCi7/KBlb1r2JhBIyB5kOns4sR3vLVl73d8fryTbsR3biTxi6/M8fuLo3vveV4l0z3vec873PJwJT42Bp8fCwxnii/jUKCFf/MVvRWrhzo9FD7Avb4fXz4Onx4t0j3Bx8EexuykpRSrKse6+6KKFcwZCActWLZop/npZpLbsOCjcKv6vE60O4q45St55B1QGFBlTcpoWO2qtkthU4SRVFzdimDIVAPtPP7W63lNejux2I6nVWFxC5SoutW3HDEAXpUapavsRZYzTotYq8ftlzFUOlPHx6GXxLLIUF7U8+fTfCxng4jVi4ea2iw2M/10d6BMWA5e/JYRCjoZCKTZJQCizdQVrtai/AREViBB2IrapH5F3pmgToVCL9N/PF4Lf3/a5fp+IYn++UEQCxl8JV38Qka8fYPi8HvavWw0cfxpjkOGnCsesaPtW7OaGsIzZ7WScJFLvJaUQvaovFC1fZD+M+xncur7rGU0RepQuV9YuWrSIefPmsWvXLrxeL8888ww7d+5k7dq1LF9+guwqnKjknAa3rBayqFveEUbI3Sh+miMpRX1X4lDR+DImQ/TKqNkPlTugsVw4bb/68diLq4P4PPBNQC1yyi9Fn4vjYdpNQuwDGUacLxy9o0mcRzgq5qpKzNWVSLJMWmY22sHHpiDmcfuoLRXRptTBLXehk7KN1JfbqC5uJHWqUCx0bNqE7PUiNVPFCyoyqgdlY65yAO1HzDpCkiTi0qKoKrRQX24jPi0Kg8kE+GksL2t5ctp
```

Comment:
the complete tutorial is without any comment what can be interpreted from the calculated and plotted results. Eventhough it is time consuming and information can be found in the PhD, a few short comments would be great, I think.